tests/libxc/dft-water/input.dat

```diff
@@ -0,0 +1,156 @@
+#  'MN12-L':   -76.3121263959,        #TEST
+#  'B3TLAP':   -76.1762459451,        #TEST
+# 'MPW1LYP':   -76.3515538422,        #TEST
+#'SOGGA11-X':  -76.3579662706,        #TEST
+#    'VSXC':   -76.4031806261,        #TEST
+# 'MPW1PBE':   -76.3399258699,        #TEST
+#   'SOGGA':   -75.9775522776,        #TEST
+#    'SCAN':   -76.3525050753,        #TEST
+#'MPW1PW91':   -76.3639997875,        #TEST
+#     'BMK':   -76.3477995384,        #TEST
+#  'MN15-L':   -76.3003777073,        #TEST
+#  'PBEsol':   -76.0376457772,        #TEST
+#   'SCAN0':   -76.3393392953,        #TEST
+#'MGGA_MVS':   -76.3432912834,        #TEST
+#'MGGA_MVSh':  -76.3375136868,        #TEST
+#  'wM05-D':    -76.355280694,        #TEST
+#  't-HCTH':   -76.3695334542,        #TEST
+# 'BP86VWN':   -76.3897099046,        #TEST
+}                                     #TEST
+
+# The following data is from an older 
+# version of Psi4 (~April 2017, SHA: 53e752c)  
+old_psi4_data = {                     #TEST 
+   'BP86': -76.38528296852736,        #TEST
+   'BLYP': -76.36495922440312,        #TEST
+ 'B97-D3': -76.34489583201652,        #TEST
+ 'B3LYP5': -76.34795958728918,        #TEST
+  'wB97X': -76.36677266785263,        #TEST
+    'PBE': -76.29833784290976,        #TEST
+ 'M05-2X': -76.36858376817408,        #TEST
+'wB97X-D': -76.36256087667938,        #TEST
+   'wB97':  -76.3680050769001,        #TEST
+'HCTH120': -76.37568880282406,        #TEST
+   'dlDF':  -76.8428069903893,        #TEST
+   'PBE0': -76.30113068157736,        #TEST
+  'B97-1': -76.36001872648792,        #TEST
+   'PW91': -76.35555307829662,        #TEST
+  'B97-D': -76.34490642131334,        #TEST
+    'M05':  -76.3495817363072,        #TEST
+  'B3LYP': -76.38509296980465,        #TEST
+'HCTH407': -76.37515280900291,        #TEST
+'HCTH147': -76.37958046443578,        #TEST
+  'SOGGA': -75.97737386477169,        #TEST
+  'B97-2': -76.35924563701468         #TEST
+}                                     #TEST
+
+for func in qchem_data.keys():                                                                        #TEST
+    psi4_energy = energy(func, molecule=h2o)                                                          #TEST
+                                                                                                      #TEST 
+    qchem_energy = qchem_data[func]                                                                   #TEST
+                                                                                                      #TEST 
+    compare_values(qchem_energy, psi4_energy, 4, func + ': Psi4 vs. Q-Chem')                          #TEST
+                                                                                                      #TEST
+    if func in old_psi4_data:                                                                         #TEST
+            old_psi4_energy = old_psi4_data[func]                                                     #TEST
```

Comment:
Watch your formatting, this should only be indented 4 spaces.

Reply:
Good catch!